README.md

```diff
@@ -59,13 +59,21 @@ state transitions (if they want)
 * Performance benefits from using PLONK arithmetization (including 
 lookups)
 
-### Taiga overview & specification
+### Taiga overview 
 
```

Comment:
I would probably add a short explanation here like "If you want to learn more about how Taiga works, check out the sections below:
.. list of sections ..

or run the Taiga book:
```
cd book
mdbook serve --open
```

Reply:
Thanks!